linerate/equations/cigre207/convective_cooling.py

```diff
@@ -0,0 +1,339 @@
+        :math:`\text{Pr}`. The Prandtl number.
+    """
+    return 0.715 - 2.5e-4 * film_temperature
+
+
+def compute_reynolds_number(
+    wind_speed: MeterPerSecond,
+    conductor_diameter: Meter,
+    kinematic_viscosity_of_air: SquareMeterPerSecond,
+    relative_air_density: Unitless,
+) -> Unitless:
+    r"""Compute the Reynolds number using the conductor diameter as characteristic length scale.
+
+    Defined on page 5 of :cite:p:`cigre207`.
+    This is a non-standard definition which seems to indicate that the kinematic viscosity has to
+    be corrected for the density.
+
+    Parameters
+    ----------
+    wind_speed:
+        :math:`v~\left[\text{m}~\text{s}^{-1}\right]`. The wind speed.
+    conductor_diameter:
+        :math:`D~\left[\text{m}\right]`. Outer diameter of the conductor.
+    kinematic_viscosity_of_air:
+        :math:`\nu_f~\left[\text{m}^2~\text{s}^{-1}\right]`. The kinematic viscosity of air.
+    relative_air_density:
+        :math:`\rho_r~1`. The air density relative to density at sea level.
+
+    Returns
+    -------
+    Union[float, float64, ndarray[Any, dtype[float64]]]
+        :math:`\text{Re}`. The Reynolds number.
+    """
+    v = wind_speed
+    D = conductor_diameter
+    nu_f = kinematic_viscosity_of_air
+    rho_r = relative_air_density
+    return rho_r * v * D / nu_f
+
+
+## Nusselt number calculation
+#############################
+
+
+def _check_perpendicular_flow_nusseltnumber_out_of_bounds(reynolds_number):
+    Re = reynolds_number
+    if np.any(Re < 0):
+        raise ValueError("Reynolds number cannot be negative.")
+
+    if np.any(np.logical_or(Re < 100, Re > 5e4)):
+        warnings.warn("Reynolds number is out of bounds", stacklevel=5)
+
+
+def _compute_perpendicular_flow_nusseltnumber(
+    reynolds_number: Unitless,
+    conductor_roughness: Meter,
+) -> Unitless:
+    # From table on page 6 in Cigre207
+    Re = reynolds_number
+    Rs = conductor_roughness
+    conditions = [Re < 100, Re < 2.65e3, Rs <= 0.05]
+    B_choices = [0, 0.641, 0.178]
+    n_choices = [0, 0.471, 0.633]
+    B = np.select(conditions, B_choices, default=0.048)
+    n = np.select(conditions, n_choices, default=0.800)
+    return B * Re**n  # type: ignore
+
+
+def compute_perpendicular_flow_nusseltnumber(
+    reynolds_number: Unitless,
+    conductor_roughness: Meter,
+) -> Unitless:
+    r"""Compute the Nusselt number for perpendicular flow.
+
+    The Nusselt number is the ratio of conductive heat transfer to convective heat transfer.
+
+    Parameters
+    ----------
+    reynolds_number:
+        :math:`\text{Re}`. The Reynolds number.
+    conductor_roughness:
+        :math:`\text{Rs}`. The roughness number
+
+    Returns
+    -------
+    Union[float, float64, ndarray[Any, dtype[float64]]]
+        :math:`\text{Nu}_{90}`. The perpendicular flow Nusselt number.
+    """
+    _check_perpendicular_flow_nusseltnumber_out_of_bounds(reynolds_number)
+    return _compute_perpendicular_flow_nusseltnumber(
```

Comment:
Fortsatt en mulighet å slå sammen de to funksjonene (_compute_perpendicular_flow_nusseltnumber og compute_perpendicular_flow_nusseltnumber)